ui/raidboss/data/02-arr/trial/titan-hm.js

```diff
@@ -52,6 +52,7 @@
       infoText: function(data, matches) {
         return {
           en: 'Cleanse ' + data.ShortName(matches.target),
+          fr: 'Guérison ' + data.ShortName(matches.target),
           cn: '康复' + data.ShortName(matches.target),
```

Comment:
@quisquous, Was the space in English voluntary, or should it not be there?

Reply:
The space is necessary in English. If it were not there, this trigger would notify an English-language user as "CleanseVan", rather than the correct "Cleanse Van". The space should be included for the French localization.

Reply:
OK, thank you for your explanation.